crates/model/src/fee_policy.rs

```diff
@@ -0,0 +1,34 @@
+use {
+    number::serialization::HexOrDecimalU256,
+    primitive_types::U256,
+    serde::{Deserialize, Serialize},
+    serde_with::serde_as,
+};
+
+#[serde_as]
+#[derive(PartialEq, Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum FeePolicy {
+    #[serde(rename_all = "camelCase")]
+    Surplus { factor: f64, max_volume_factor: f64 },
+    #[serde(rename_all = "camelCase")]
+    Volume { factor: f64 },
+    #[serde(rename_all = "camelCase")]
+    PriceImprovement {
+        factor: f64,
+        max_volume_factor: f64,
+        quote: Quote,
+    },
+}
```

Comment:
Will open a follow-up PR shortly to get rid of duplicated `FeePolicy` struct declarations.

Reply:
Which ones? I think the autopilot domain definition and orderbook domain (which we don't really have yet) should be separate structs (we may want to expose different fee information on the API one day).

Reply:
Yeah, that is why there is still no PR. Initially, I thought a model crate could be used for those, but it looks like this is not the case.